pkg/ingester/transfer.go

```diff
@@ -98,8 +98,13 @@ func (i *Ingester) TransferChunks(stream logproto.Ingester_TransferChunksServer)
 
 		instance := i.getOrCreateInstance(chunkSet.UserId)
 		for _, chunk := range chunkSet.Chunks {
-			if err := instance.consumeChunk(userCtx, lbls, chunk); err != nil {
-				return err
+			err := instance.consumeChunk(userCtx, lbls, chunk) err != nil {
+				if err.Error() == "invalid checksum" {
+					level.Error(util.Logger).Log("msg", "Checksum does not match for chunk, this chunk will be skipped", "err", "invalid checksum", "chunk", chunk.String())
```

Comment:
Not sure if logging `chuck.String()` is a good idea

Reply:
I am not able reproduce this issue(#1346). Can we request the author of this issue to validate from his side using this commit and see what he sees in his logs?

Reply:
I don't think this change is necessary since now you're not sending back the invalid error.

Reply:
Let's leave the transfer erroring (rollback that file), I don't think this is causing any issue for #1346.